FirebaseStorage/Sources/Storage.swift

```diff
@@ -97,19 +97,31 @@ import FirebaseAuthInterop
    * The maximum time in seconds to retry an upload if a failure occurs.
    * Defaults to 10 minutes (600 seconds).
    */
-  @objc public var maxUploadRetryTime: TimeInterval
```

Comment:
I missed this adjustment in the first port. The Objective C implementation was also synchronized on `self`, but I'm not sure how to do that in Swift or if it's needed.